vsts/determine_tests_to_run.ps1

```diff
@@ -26,6 +26,10 @@ ForEach ($line in $($GitDiff -split "`r`n"))
 	{
         # These file types are ignored when determining if source code changes require running e2e tests
 	}
+	elseif ($line.toLower().Contains("sample"))
+	{
+	    # Sample changes don't necessitate running e2e tests
```

Comment:
What about changes to scripts, yaml files, versions.csv etc

Reply:
Those changes wouldn't be in a folder that contains "iothub", "provisioning", etc so the script already disregards them

Reply:
cool!